schemas/product.js

```diff
@@ -19,7 +19,7 @@ let Product = mongoose.model("Product", {
   },
   ESG: {
     type: String,
-    // Optional by default
+    requred: true
```

Comment:
Why did we change this back to required? Is that because we are setting it to 0 by default now?

Reply:
No product should be entered into our database without an ESG score if the app is built around ESG scores.

Reply:
Well I think its fine to have things in there that we fill in later, but we can't expect customers to have access to ESG when they are manually entering (and we may only have access to a subset of ESG data available for the time being, which we may fill in later). Setting it to 0 is fine though.